### PR TITLE
Documented searching

### LINKED ISSUE #19: Document commit searching (#586)
Document https://github.com/versionpress/versionpress/issues/586.

### PR DESCRIPTION
**Pull request by borekb**
Resolves #19 

With some presumed changes that are not yet done in https://github.com/versionpress/versionpress/pull/730. 

Reviewers:
- [x] @vasek17

### CODE REVIEW

**Comment by borekb** [comment]
@vasek17 Could you please review this PR whether the search description is technically accurate? Thank you.

File: content/en/02-feature-focus/04-searching-history.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+All of the syntaxes above can be freely combined.
+
+
+## Operators
+
+### `author:`
+
+Author of the action. You can use author name or his/her email, wildcards are supported.
+
+Anonymous actions like posting a comment on a blog are tracked under a virtual user `nonadmin@example.com`.
```
Comment (vasek) [inline content/en/02-feature-focus/04-searching-history.md]: Maybe mention also WP-CLI user?

File: content/en/02-feature-focus/04-searching-history.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+
+VP tags are pieces of metadata that VersionPress stores with each commit. For example, updating the site title creates a commit message like this:
+
+```
+[VP] Edited option 'blogname'
+
+VP-Action: option/edit/blogname
+
+X-VP-Version: 3.0
+X-VP-Environment: master
```
Comment (vasek) [inline content/en/02-feature-focus/04-searching-history.md]: Change `master` to `default` as it is the new default env name?

File: content/en/02-feature-focus/04-searching-history.md (hunk 1 above)
Comment (borekb) [inline content/en/02-feature-focus/04-searching-history.md]: Good point, done.

File: content/en/02-feature-focus/04-searching-history.md (hunk 2 above)
Comment (borekb) [inline content/en/02-feature-focus/04-searching-history.md]: Changed this to `staging` which is probably even a better name (for docs).

**Pull request merged**